jsonschema-core.xml

```diff
@@ -2727,18 +2726,18 @@
                             Basic - Provides validation information in a flat list structure.
                         </t>
                         <t>
-                            Detailed - Provides validation information in a condensed hierarchical
-                            structure based on the structure of the schema.
-                        </t>
-                        <t>
-                            Verbose - Provides validation information in an uncondensed hierarchical
-                            structure that matches the exact structure of the schema.
+                            Hierarchical - Provides validation information in a hierarchical
+                            structure that mimics the structure of the schema.
+                            <cref>
+                                To be precise, this structure follows the paths that a validator
+                                would traverse while processing the schema, including composition
+                                via by-reference keywords.
+                            </cref>
```

Comment:
Can we say "that matches the evaluation paths of the schema" or something like that?  Since we use `evaluationPath` as the name for identifying points within this hierarchical process.  Perhaps "evaluation structure"?  This would let you drop the CREF.

I also like the idea of talking about an "evaluation structure" because that provides context for the `evaluationPath` JSON Pointers.  There have been a few complaints about these pointers not corresponding to a document, which is not entirely true:  they do not correspond to a _persistent_ document, but they do correspond to the document structure traced by evaluation.  Naming that structure will make it easier to talk about that correspondence.

Reply:
"dereferenced evaluation structure"?

Reply:
> "dereferenced evaluation structure"?

That could work as long as it's defined somewhere.

Or maybe "dynamic scope document" which might not be as intuitive, but if defined clearly would fit with focusing on dynamic scopes?  idk, just tossing this out there, don't feel strongly about it.

Reply:
I think going with 

> Provides validation information in a hierarchical structure that _follows the evaluation paths generated while processing the schema_.

Aligns better with my newest (marginally related) PR, #1264.